central/graphql/resolvers/cluster_vulnerabilities.go

```diff
@@ -312,10 +312,9 @@ func withOpenShiftTypeFiltering(q string) string {
 		search.NewQueryBuilder().AddExactMatches(search.CVEType, storage.CVE_OPENSHIFT_CVE.String()).Query())
 }
 
-func (resolver *clusterCVEResolver) clusterVulnerabilityScopeContext() context.Context {
+func (resolver *clusterCVEResolver) clusterVulnerabilityScopeContext(ctx context.Context) context.Context {
 	if resolver.ctx == nil {
-		log.Errorf("attempted to scope context on nil")
```

Comment:
😱

Reply:
Do we want to retain these messages? They don't seem necessary or at the very least shouldn't be errors